src/services/w3c/verifier.rs

```diff
@@ -370,6 +370,24 @@ fn check_request_data(
         )?;
     }
 
+    for (cred, proof) in presentation
```

Comment:
Should these checks also be made when calling credential_process?

Otherwise looks good to me, thanks for picking this up

Reply:
Yes, there might be additional verification required for w3c credentials, I was only looking at presentations.